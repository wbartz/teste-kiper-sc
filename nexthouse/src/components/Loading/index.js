import PropTypes from 'prop-types';
import React, { useContext } from 'react';
import { Loader } from 'react-overlay-loader';
import { AppContext } from '../../containers/StoreProvider';

const Loading = ({ isLoading }) => {
  return <Loader loading={isLoading} fullPage text="" />;
};

Loading.propTypes = {
  isLoading: PropTypes.bool,
};

Loading.defaultProps = {
  isLoading: false,
};

export default props => <Loading {...useContext(AppContext)} {...props} />;
