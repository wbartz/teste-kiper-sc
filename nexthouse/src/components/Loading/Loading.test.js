import { shallow } from 'enzyme';
import React from 'react';
import Loading from '.';

describe('Loading', () => {
  let wrapper;
  const props = {
    isLoading: false,
  };

  beforeEach(() => {
    wrapper = shallow(<Loading {...props} />); // eslint-disable-line
  });

  it('render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
