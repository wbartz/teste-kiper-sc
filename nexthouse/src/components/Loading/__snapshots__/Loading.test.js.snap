// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading render without crashing 1`] = `ShallowWrapper {}`;
